index.css

```diff
@@ -66,4 +66,25 @@ body {
     color: #ffffff;
     background-color: #10859e;
     border-color: #0e758b;
-}
\ No newline at end of file
+}
+
+#uv-container {
+    position: relative;
+}
+
+#logout {
+    top: 0;
+    right: 0;
+    position: absolute;
+    padding: 10px;
```

Comment:
We could move the padding to the `a` tag to give that extra bit of room for the link to register

Reply:
I tried this but then the padding seemed not to take effect.

Reply:
`display: inline-block` should do the trick on the a tag